telegram/README.md

```diff
@@ -0,0 +1,37 @@
+# Telegram Bot Command
+
+A simple Telegram Bot Command.
+
+## Tutorial
+
+- Follow the
```

Comment:
Maybe we should switch to using numbered lists as these are step-by-step instructions.
```suggestion
1. Follow the
```